### PR TITLE
Add Tests

### PR DESCRIPTION
**Pull request by hemslo**
Split app and env code out so it can be tested, currently only covers non keycloak logic.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        callbackURL: `${baseUriPath}/api/auth/callback`,
+        authorizationURL: `${KC_HOST}/realms/${KC_REALM}/protocol/openid-connect/auth`,
+        tokenURL: `${KC_HOST}/realms/${KC_REALM}/protocol/openid-connect/token`,
+        userInfoURL: `${KC_HOST}/realms/${KC_REALM}/protocol/openid-connect/userinfo`
+      },
+
+      async (accessToken, refreshToken, profile, done) => {
+        console.log(`username: ${profile.username} roles: ${profile.roles}`)
+
+        const getRole = (userRoles) => {
```
Comment (janboll) [inline app.js:37]: Is it worth splitting this out and adding a test for getRole() as well?

**Review by janboll — APPROVED** [review]
Looks great, thank you!

File: app.js (hunk 1 above)
Comment (hemslo) [reply app.js:37]: Yes, I can, just this test won't be the same as other tests using request, I haven't figured out how to test keycloak part properly.

Comment (hemslo) [reply app.js]: added in https://github.com/app-sre/unleash/pull/57/commits/ace23ed79dfaaf9eefdeacd5de28be302c4c58f5

**Pull request merged**